examples/scheduling.rst

```diff
@@ -0,0 +1,245 @@
+  (under 30 minutes) and :math:`0` for meetings of longer duration.
+* Participation is represented by variable :code:`mandatory` with value :math:`1` for mandatory
+  participation and :math:`0` for optional participation.
+
+For large numbers of variables and constraints, such problems can be hard.
+This example has four binary variables, so only :math:`2^4=16` possible meeting arrangements.
+As shown in the table below, it is a simple matter to work out all the combinations by hand to
+find solutions that meet all the constraints.
+
+.. table:: All Possible Meeting Options.
+   :name: MeetingOptions
+
+   ====================  =================  ==============  ==================  =================
+   **Time of Day**       **Venue**          **Duration**    **Participation**   **Valid?**
+   ====================  =================  ==============  ==================  =================
+   Business hours        Office             Short           Mandatory           Yes
+   Business hours        Office             Short           Optional            No (violates 2)
+   Business hours        Office             Long            Mandatory           Yes
+   Business hours        Office             Long            Optional            No (violates 2)
+   Business hours        Teleconference     Short           Mandatory           No (violates 1)
+   Business hours        Teleconference     Short           Optional            No (violates 1, 2)
+   Business hours        Teleconference     Long            Mandatory           No (violates 1)
+   Business hours        Teleconference     Long            Optional            No (violates 1, 2)
+   Non-business hours    Office             Short           Mandatory           No (violates 3)
+   Non-business hours    Office             Short           Optional            No (violates 3)
+   Non-business hours    Office             Long            Mandatory           No (violates 3, 4)
+   Non-business hours    Office             Long            Optional            No (violates 3, 4)
+   Non-business hours    Teleconference     Short           Mandatory           Yes
+   Non-business hours    Teleconference     Short           Optional            Yes
+   Non-business hours    Teleconference     Long            Mandatory           No (violates 4)
+   Non-business hours    Teleconference     Long            Optional            No (violates 4)
+   ====================  =================  ==============  ==================  =================
+
+Ocean's `dwavebinarycsp <http://dwavebinarycsp.readthedocs.io/en/latest/>`_ enables the
+definition of constraints in different way, including by defining functions that evaluate
+True when the constraint is met. The code below defines a function that returns True when
+all this example's constraints are met.
+
+.. code-block:: python
+
+   >>> def scheduling(time, location, length, mandatory):
+       if time:                                 # Business hours
```

Comment:
Incorrect indent here. Furthermore I *think* that `.. code-block:: python` is used without the `>>>`. Using both is redundant. The difference is the `>>>` get picked up by doctest and are therefore preferred.